apps/kol-amelamdim/components/upload-file-dialog/UploadFileDialog.tsx

```diff
@@ -164,7 +169,13 @@ export const UploadFileDialog = ({
           <MenuItem value={Category.shonot}>שונות</MenuItem>
         </Select>
       </FormControl>
-
+      <TextField
```

Comment:
Let's put this textfield first before category

Reply:
Resolved.